catalog/imaging/catalog-video-transformer/src/test/java/com/connexta/transformer/video/TestListOfDoublesHandler.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) Connexta, LLC
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.transformer.video;
+
+import static org.hamcrest.Matchers.closeTo;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+import org.codice.ddf.libs.klv.KlvDecodingException;
+import org.codice.ddf.libs.klv.data.numerical.KlvInt;
+import org.junit.Test;
+
+public class TestListOfDoublesHandler {
+
+    @Test
+    public void testEmpty() {
+        KlvHandler handler = new ListOfDoublesHandler("field");
+        assertThat(handler.asAttribute()
+                .isPresent(), is(false));
+    }
+
+    @Test
+    public void testAcceptWrongType() {
+        KlvHandler handler = new ListOfDatesHandler("field");
```

Comment:
`ListOfDoublesHandler`? You could create an instance variable `handler` and instantiate it in a `@Before` method so you don't have to do it manually for every test.